src/axom/core/ArrayView.hpp

```diff
@@ -16,6 +16,12 @@ namespace axom
 template <typename T, int DIM, MemorySpace SPACE>
 class ArrayView;
 
+template <typename T, int DIM, MemorySpace SPACE>
+struct ArrayTraits<ArrayView<T, DIM, SPACE>>
+{
+  constexpr static bool IsView = true;
```

Comment:
I agree with @joshessman-llnl .  I'm pretty sure we only use camel-casing for classes. I would prefer pot-hole casing.

@kennyweiss ?

Reply:
Yes, we use camelCaseForClassesAndMethods. Typically, we use pot-hole case for variable names.